charts/shoot-core/components/charts/vpn-shoot/templates/_helpers.tpl

```diff
@@ -1,9 +1,10 @@
+#doubt
 {{- define "vpn-shoot.certs.data" -}}
 ca.crt: {{ .Values.vpnShootSecretData.ca }}
 tls.crt: {{ .Values.vpnShootSecretData.tlsCrt }}
 tls.key: {{ .Values.vpnShootSecretData.tlsKey }}
 {{- end -}}
-
+#doubt
 {{- define "vpn-shoot.certs.name" -}}
 vpn-shoot-{{ include "vpn-shoot.certs.data" . | sha256sum | trunc 8 }}
```

Comment:
should we do the checksum and trunc beforehand and save it in a variable to use while creating the object?

Reply:
This is not needed now anymore since there is a helper function that you can call in the Golang code, see https://github.com/shafeeqes/gardener/blob/refactor/vpn-shoot/pkg/operation/botanist/component/metricsserver/metrics_server.go#L415